listenbrainz/webserver/views/explore.py

```diff
@@ -9,51 +9,21 @@
 explore_bp = Blueprint('explore', __name__)
 
 
-@explore_bp.route("/")
-def index():
-    """ Main explore page for users to browse the various explore features """
-
-    return render_template(
-        "explore/index.html",
-        props=orjson.dumps({}).decode("utf-8")
-    )
-
-
-@explore_bp.route("/huesound/")
-def huesound():
-    """ Hue Sound browse music by color of cover art """
-
-    return render_template(
-        "explore/huesound.html",
-        props=orjson.dumps({}).decode("utf-8")
-    )
-
-
-@explore_bp.route("/similar-users/")
+@explore_bp.route("/similar-users/", methods=['POST'])
 def similar_users():
     """ Show all of the users with the highest similarity in order to make
         them visible to all of our users. This view can show bugs in the algorithm
         and spammers as well.
     """
 
     similar_users = get_top_similar_users()
-    return render_template(
-        "explore/similar-users.html",
-        similar_users=similar_users
-    )
 
-
-@explore_bp.route("/fresh-releases/")
-def fresh_releases():
-    """ Explore fresh releases """
-
-    return render_template(
-        "explore/fresh-releases.html",
-        props=orjson.dumps({}).decode("utf-8")
-    )
+    return jsonify({
+        "similarUsers": similar_users
+    })
 
 
-@explore_bp.route("/music-neighborhood/")
+@explore_bp.route("/music-neighborhood/", methods=['POST'])
```

Comment:
Don't know how relevant it is, but in some places (specifically API endpoints IIRC) we've had to add the `OPTIONS` method.
No idea if that would be needed here for some reason or not, just voicing it just in case.